posts/dask-summit-takeaway.md

```diff
@@ -0,0 +1,120 @@
+  - Unlock the community - entrain users, become contributors
+- **_Provide scope and build people_**
+  - Move towards "grassroots" not so much top down
+
+## Xarray User Forum + Updates
+
+- Xarray now includes a backend api, allowing contributors to add new backends for reading + operating on datasets
+  - Including the `raw_indexing` method is important to enable lazy loading
+- Flexible indexing is coming to xarray soon
+  - Helps when dealing with irregular spaced data, working with projected coordinate systems, staggered grids
+  - Currently, Xarray uses pandas for indexing, which is problematic when wanting to do flexible indexing
+  - Current Xarray accessor - [Xoak](https://xoak.readthedocs.io/en/latest/)
+    - See [previous blog post on working with unstructured grids](https://ncar.github.io/esds/posts/multiple_index_xarray_xoak/)
+  - See the [design documents for flexible indexing in Xarray](https://github.com/pydata/xarray/blob/master/design_notes/flexible_indexes_notes.md)
+- Duck arrays are now supported in Xarray
+  - Helps when working with [pint](https://pint.readthedocs.io/en/stable/)
+  - **_Most_** of the xarray API supports duck arrays, few aspects that still don't
+
+### [Xarray User Forum Talk Links](https://summit.dask.org/schedule/presentation/51/xarray-user-forum/)
+
+- Aureliana Barghini’s Talk
+
+  - [New backend plugin infrastructure](https://github.com/aurghs/xarray-backend-tutorial)
+
+- Links from Benoit Bovy’s talk
+
+  - [Flexible indexes design notes](https://github.com/pydata/xarray/blob/master/design_notes/flexible_indexes_notes.md)
+  - [Explicit indexes project](https://github.com/pydata/xarray/projects/1)
+
+- Davis Bennett’s talk
+
+  - [Hierarchical storage data structure in xarray](https://github.com/pydata/xarray/issues/4118)
+
+- Ravi Kumar’s talk:
+  - [Using Xarray to store statistical diagnostic data](https://github.com/arviz-devs/arviz_misc/tree/master/xarray_user_forum_2021)
+- Jack’s talk:
+  - [Xarray Fusion Research](https://xyzpy.readthedocs.io/en/latest/)
+
+## How can we tie this back to ESDS?
+
+Several of these sessions included speakers/organizers from within NCAR, such as Anderson Banihirwe [(@andersy005)](https://github.com/andersy005) and Deepak Cherian [(@dcherian)](https://github.com/dcherian). They, along with several others with NCAR, could be considered the "passionate few". They are core developers of upstream projects such as Xarray and Dask, which are critical components to the scientific python ecosystem, especially for geoscience. The key to ESDS is growing the userbase, moving toward the "productive many".
+
+We should see the ESDS initiative as an opportunity to test out these ideas, and push to forge ahead with a grassroots movement focused around promoting open-science and open-devlopment, empowering people to be more productive in their data workflows. Here are a few key points for us to consider:
+
+- Developing in small teams
```

Comment:
leveraging the existing open-source community. "humble scoping and build interfaces"